src/portfft/dispatcher/workgroup_dispatcher.hpp

```diff
@@ -282,6 +283,7 @@ struct committed_descriptor<Scalar, Domain>::run_kernel_struct<LayoutIn, LayoutO
                              const std::vector<sycl::event>& dependencies, IdxGlobal n_transforms,
                              IdxGlobal input_offset, IdxGlobal output_offset, dimension_struct& dimension_data,
                              direction compute_direction) {
+    PORTFFT_LOG_FUNCTION_ENTRY();
```

Comment:
Using a single comment for all the 4 implementations.
It would be beneficial to log the amount of local memory being used in all the implementations

Reply:
This is already logged. For workgroup implementation see line 319.

Reply:
I missed that it was with the global and local range for WI/SG/WG.
I suppose it is missing from the global level though. In `compute_level` we can log `loc_for_twiddles`, `loc_for_input` and `loc_for_modifiers`.

Reply:
Adding to this comment, logging of global/local range can be logged inside `compute_level` as well

Reply:
@t4c1 Did you perhaps miss this comment ?

Reply:
I did. Done now.